react-tutorial/src/components/Caption.css

```diff
@@ -0,0 +1,6 @@
+.Caption {
+    width: 25%;
```

Comment:
Ideally, such styling properties that refer to fixed size should be used in outer node - this will allow to have a component itself more flexible in displaying.